thirdparty/vars.sh

```diff
@@ -207,7 +207,7 @@ ROCKSDB_MD5SUM="02727e52cdb94fa6a9dbbd68d157e619"
 # libsasl
 SASL_DOWNLOAD="https://github.com/cyrusimap/cyrus-sasl/archive/refs/tags/cyrus-sasl-2.1.28.tar.gz"
 SASL_NAME=cyrus-sasl-2.1.28.tar.gz
-SASL_SOURCE=cyrus-sasl-2.1.28
+SASL_SOURCE=cyrus-sasl-cyrus-sasl-2.1.28
```

Comment:
what's wrong, why need these changes?

Reply:
In download-thirdparty.sh 222-224, it creates a new folder if there is no folder extracted from the zip file. Then, I adjust the SOURCE name to meet with the one extracted from the zip file.